.tx/config

```diff
@@ -1,7 +1,7 @@
 [main]
 host = https://www.transifex.com
 
-[o:cataclysm-dda-translators:p:cataclysm-dda:r:master-cataclysm-dda]
+[o:cataclysm-dda-translators:p:cataclysm-dda:r:0.H-cataclysm-dda]
```

Comment:
Turns out this needs to be the encoding-friendly "slug" instead of the name.
```suggestion
[o:cataclysm-dda-translators:p:cataclysm-dda:r:0h-cataclysm-dda]
```